src/app/components/timepicker/index.html

```diff
@@ -10,28 +10,37 @@
     specified on the <stache-code>input</stache-code> element.
   </sky-demo-page-summary>
 
-  <sky-demo-page-properties>
+  <sky-demo-page-properties sectionHeading="Timepicker input directive properties">
     <sky-demo-page-property
-      propertyName="timeFormat"
-      isOptional="true"
-      defaultValue="hh"
+      propertyName="skyTimepickerInput"
     >
-    Specifies 12-hour <stache-code>hh</stache-code> or 24-hour <stache-code>HH</stache-code> format.
+    Creates the timepicker input and dropdown. Place this attribute on an <stache-code>input</stache-code> element, and wrap the input in a <stache-code>sky-timepicker</stache-code> component. This attribute must be set to the instance of the <stache-code>sky-timepicker</stache-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="disabled"
+      isOptional="true"
+      defaultValue="false">
+    Indicates whether to disable the datepicker. This property accepts <stache-code>boolean</stache-code> values.
```

Comment:
"Indicates whether to disable the datepicker." should be "Indicates whether to disable the timepicker."

Reply:
Good catch. Thanks.